src/pages/EnablePayments/OnfidoStep.js

```diff
@@ -0,0 +1,154 @@
+import React from 'react';
+import {View, Text} from 'react-native';
+import {withOnyx} from 'react-native-onyx';
+import PropTypes from 'prop-types';
+import Onfido from '../../components/Onfido';
+import FullscreenLoadingIndicator from '../../components/FullscreenLoadingIndicator';
+import ONYXKEYS from '../../ONYXKEYS';
+import {activateWallet, fetchOnfidoToken} from '../../libs/actions/BankAccounts';
+import Navigation from '../../libs/Navigation/Navigation';
+import CONST from '../../CONST';
+import Button from '../../components/Button';
+import HeaderWithCloseButton from '../../components/HeaderWithCloseButton';
+import styles from '../../styles/styles';
+import TextLink from '../../components/TextLink';
+import withLocalize, {withLocalizePropTypes} from '../../components/withLocalize';
+import compose from '../../libs/compose';
+
+const propTypes = {
+    /** Stores various information used to build the UI and call any APIs */
+    walletOnfidoData: PropTypes.shape({
+
+        /** Unique identifier returned from fetchOnfidoToken then re-sent to ActivateWallet with Onfido response data */
+        applicantID: PropTypes.string,
+
+        /** Token used to initialize the Onfido SDK token */
+        sdkToken: PropTypes.string,
+
+        /** Loading state to provide feedback when we are waiting for a request to finish */
+        loading: PropTypes.bool,
+
+        /** Error message to inform the user of any problem that might occur */
+        error: PropTypes.string,
+
+        /** Whether the user has accepted the privacy policy of Onfido or not */
+        hasAcceptedPrivacyPolicy: PropTypes.bool,
+    }),
+
+    ...withLocalizePropTypes,
+};
+
+const defaultProps = {
+    walletOnfidoData: {
+        loading: false,
+        hasAcceptedPrivacyPolicy: false,
+    },
+};
+
+class OnfidoStep extends React.Component {
+    canShowOnfido() {
```

Comment:
Missing method doc

Reply:
It's OK to omit method docs if it's self explanatory, there are no params, and there are no return statements (same as with PHP). However, I guess this does have a return statement, so I'll go ahead and document that!